### DISCUSSION TITLE
Screen modes - feature request

### DISCUSSION THREAD

**Discussion by gobonja** [tribalfs/GalaxyMaxHzPub#131] [Ideas]
Samsung has more than 2 screen modes but they can only be changed using adb:
https://www.reddit.com/r/Galaxy_S20/comments/jlid2s/vivid_and_natural_are_not_the_only_screen_modes/

Is it possible to support this through GalaxyMaxHz?

Thank in advance

**Comment by tribalfs** [comment]
It's possible. But the screen mode setting is available in device settings in Settings > Display > Screen mode

**Comment by semool** [comment]
@gobonja is right.
In the Settings you can only select Natural and Vivid.

There is 5 screen modes in total:

0 AMOLED cinema
1 AMOLED Photo
2 Basic
3 Natural
4 Vivid

Its explained in the Reddit Link.
